src/flowchem/client/client.py

```diff
@@ -14,10 +14,28 @@
 
 
 class FlowchemDeviceListener(FlowchemCommonDeviceListener):
-    def _save_device_info(self, zc: Zeroconf, type_: str, name: str) -> None:
+    """
+    creat
+    """
+    def _save_device_info(self, zc: Zeroconf, type_: str, name: str, active_ips: list) -> None:
+        """
+
+        Args:
+            zc:
+            type_: '_labthing._tcp.local.'
+            name: 'device._labthing._tcp.local.'
+
+        Returns:
+
+        """
+        zc_info = zc.get_service_info(type_, name)
```

Comment:
What is the purpose of this variable? You do not use it anywhere; it is not an attribute either.

Reply:
sorry this a wrong commit I did, it can be delete